### DISCUSSION TITLE
Add a command named /cmi shell to execute os commands

### DISCUSSION THREAD

**Discussion by sakurawald** [Zrips/CMI#8100] [Feature Requests]
Add a command like /cmi shell [args] 
This is very useful like.
You can execute some other programs with in-game args (placeholder).
You can choose to run in sync or async mode.

1. cp ./template/paper-world.yml ./resource_world/
2. curl and webhook
3. execute some other programs with [args]

**Comment by MrCrazys** [comment]
While this could be useful, it also comes with various security risks. If someone hacked your server, they would be able to execute shell commands from in the game. Besides, it seems a bit out of scope for CMI.

**Reply by CloudeLecaw** [reply]
i totally agree with you. Itsound very dangerous in view about stafty security and is a bit out of the scrope of cmi.
